src/Card.js

```diff
@@ -19,19 +19,40 @@ export default class Card extends Component {
 
 
   render() {
+    console.log('starring', this.props.episode.starring)
```

Comment:
I like that you are checking to see what information you are getting, but we may want to remove this console.log before merging it all in so we don't miss it later.